core/app/flags/experimental.go

```diff
@@ -23,7 +23,8 @@ import (
 //     2) be by default false/off
 //     3) be removed once the feature is no longer in experiment
 var (
-	EnableVulkanTracing = flag.Bool("experimental-enable-vulkan-tracing", false, "Enable the experimental feature Vulkan tracing.")
-	EnableAngleTracing  = flag.Bool("experimental-enable-angle-tracing", false, "Enable the experimental feature ANGLE tracing.")
-	EnablePerfTab       = flag.Bool("experimental-enable-perf-tab", false, "Enable the experimental feature Counter performance tab.")
+	EnableVulkanTracing      = flag.Bool("experimental-enable-vulkan-tracing", false, "Enable the experimental feature Vulkan tracing.")
+	EnableAngleTracing       = flag.Bool("experimental-enable-angle-tracing", false, "Enable the experimental feature ANGLE tracing.")
+	EnablePerfTab            = flag.Bool("experimental-enable-perf-tab", false, "Enable the experimental feature Counter performance tab.")
+	EnableProfileExperiments = flag.Bool("experimental-enable-profile-experiments", false, "Enable Profile Experiments.")
```

Comment:
I don't think we need this to be a server-side flag. Just make it a UI flag only for now.